src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Machine {
+
+    public static void doWork() {
```

Comment:
The method 'doWork' should not be static. Since 'Machine' will be a superclass for other specific machine types, instance methods are preferable to allow different implementations for each subclass.